test/core/unwind.wast

```diff
@@ -70,18 +92,18 @@
       (i32.const 3) (block (i64.const 1) (return (i32.const 9)))
     )
   )
-  (func (export "br_if-unwind-by-last-value") (param i32) (result i32)
-    (block (result i32)
-      (br_if 0 (i32.const 100) (i32.const 10) (get_local 0)) (return (i32.const 9))
-    )
-  )
 
   (func (export "unary-after-unreachable") (result i32)
     (f32.const 0) (unreachable) (i64.eqz)
   )
   (func (export "unary-after-br") (result i32)
     (block (result i32) (f32.const 0) (br 0 (i32.const 9)) (i64.eqz))
   )
+  ;;(func (export "unary-after-br_if") (result i32)
+  ;;  (block (result i32)
+  ;;    (f32.const 0) (drop (br_if 0 (i32.const 9) (i32.const 1))) (i64.eqz)
```

Comment:
This drops the remaining argument of the br_if, but still leaves the f32 on the stack. The i64.eqz expects an i64, though. Maybe just change the const into an i64?